CONTRIBUTING.md

```diff
@@ -35,8 +35,8 @@ Please consult the issues and discussions section of the SBT repo for good first
 1. Make sure existing (and new) tests pass successfully by running `npm run test`
 1. Run `npm run build` to compile
 1. Go to the root of the project. Then, deploy the CDK stack using the following:
-   - For `core-app-plane`: `npx cdk deploy --app='./lib/core-app-plane/integ.default.js'`
-   - For `control-plane`: `CDK_PARAM_SYSTEM_ADMIN_EMAIL="test@example.com" npx cdk deploy --app='./lib/control-plane/integ.default.js'`
+   - For `core-app-plane`: `npx cdk deploy --app='./src/core-app-plane/integ.default.js'`
```

Comment:
the src/ dir contains the typescript files that we update whereas the lib/ dir contains the output js files that are used by deployed and used by end-customers.

Reply:
reverted and added instructions to run npm install before npm run test